Client/Frontend/TabContentsScripts/CreditCardHelper.swift

```diff
@@ -62,8 +67,15 @@ class CreditCardHelper: TabContentScript {
     func userContentController(_ userContentController: WKUserContentController,
                                didReceiveScriptMessage message: WKScriptMessage) {
         guard let data = getValidPayloadData(from: message) else { return }
-        guard let payload = parseFieldType(messageBody: data)?.creditCardPayload else { return }
-        foundFieldValues?(getFieldTypeValues(payload: payload))
+        guard let fieldValues = parseFieldType(messageBody: data) else { return }
```

Comment:
nit: I think the three guards could be merged into one and it will be easier to read and understand

Reply:
Oh nice, yeah pushed a small commit here https://github.com/mozilla-mobile/firefox-ios/pull/14739/commits/0e8ce71e953d44089a6e4431919bfa0eb26386f5